.github/actions/push_to_mrt/action.yml

```diff
@@ -11,7 +12,7 @@ runs:
   steps:
     - name: Push Bundle to MRT
       run: |-
-        cd ${{ inputs.CWD }}
+        cd ${{ inputs.cwd }}
```

Comment:
Should this be `cwd` or `CWD`? I see all caps on line 3.

Reply:
In my experience, the variables in github actions are case insensitive. But as the other variables in that action are also in caps, I've changed this back to caps.